console/src/views/Dashboard.js

```diff
@@ -87,11 +93,17 @@ const DashboardBase = kind({
 						<Cell size="20%">
 							<Column className={css.sideInfo} align="stretch space-between">
 								<Cell shrink>
-									<IconItem icon="compass" label="23.1 psi" spotlightDisabled>Tire psi</IconItem>
+									<Item label="23.1 psi" spotlightDisabled>
```

Comment:
We should place the `Icon` in `slotBefore` to ensure the label and text are aligned correctly (and that the icon doesn't marquee with long text).

I changed this one but left the other as is in the screenshot below.

![image](https://user-images.githubusercontent.com/788456/68440246-45199400-017f-11ea-9eb0-0ef2d212ed53.png)